plugins/generator-1.19.4/datapack-1.19.4/jsontriggers/item_consumed.json.ftl

```diff
@@ -0,0 +1,10 @@
+{
+  "trigger": "minecraft:consume_item",
+  "conditions": {
+    "item": {
+        "items": [
+            "${input$item}"
+  		]

```

Comment:
Fix indentation in this file

Reply:
Interestingly enough I noticed this inconvenience when checking trigger files before making the commit and somehow did forget about this case...

Reply:
✔️ Fixed